com.unity.netcode.gameobjects/Runtime/Interest/InterestManager.cs

```diff
@@ -84,6 +85,39 @@ public void QueryFor(ref TObject client, ref HashSet<TObject> results)
             }
         }
 
+        public void AddInterestNode(ref TObject obj, IInterestNode<TObject> node)
```

Comment:
I discovered something odd here, which is if I try and pass `node` by `ref` the compiler will complain if I pass a subclass of `IInterestNode`